### ISSUE TITLE
commented_code_linter doesn't like gene names

### ISSUE THREAD

**Issue by pimentel** [jimhester/lintr#125]
The following comment produces a warning about commented code, though it is really just a gene name. There are many gene names like this. I imagine that warning is due to the `-`, so I'm not sure if there is anything "to fix" but I'm curious if you have any suggestions on how to deal with this (beyond simply disabling it):

code:

```{r}
# YJL133C-A
```

Warning:

```
Commented code should be removed.
```

Thanks

**Comment by jimhester** [comment]
Only comments which parse into valid R code will get linted, so the easiest way would be to make it non-valid, e.g.
```r
# Gene YJL133C-A
```

You could also turn off linting for that line
```r
# YJL133C-A # nolint
```

**Comment by pimentel** [comment]
Perfect, thanks!
